src/utils/source.js

```diff
@@ -329,3 +340,16 @@ export function getTextAtPosition(source: Source, location: Location) {
 
   return lineText.slice(column, column + 100).trim();
 }
+
+export function getSourceClassnames(
+  source: Object,
+  ...additionalClasses: Array<string>
+) {
+  const sourceRecord = new SourceRecordClass(source);
```

Comment:
why is this needed?

Reply:
to use the .get method it needs to be a SourceRecord. getExtension also uses .get even though the method signature says the source parameter is a plain object. My initial inclination was to maintain the contract there.

However, `getExtension` is only used in`getSourceClassnames`, so as far as I am concerned we should just be working with plain js objects here. Good catch.